app.py

```diff
@@ -1,12 +1,26 @@
 import os
 import time
 import gradio as gr
+import requests
 from dotenv import load_dotenv
 from griptape.chat_demo import Chat
+from griptape.chat_cloud import Chat_Cloud, Chat_Local
 
+# Load the environment variables
 load_dotenv()
+
+# Get the port from the environment variables for Gradio (default is 7860)
 port = os.getenv("GRADIO_PORT", 7860)
-chat = Chat()
+
+# Get the lambda endpoint from environment variables if using CDK based memory
+lambda_endpoint = os.getenv("LAMBDA_ENDPOINT", "")
+ 
+# Function to get the session id from the lambda endpoint
+def get_state():
```

Comment:
function is called `get_state`, but the value is actually a session ID. Suggest also using Python type hints for return value (e.g., `def get_session_id() -> Str`). (assuming it does return a String, not sure what type it returns)